lib/pages/team-details/team_detail_page.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/cupertino.dart';
+import 'package:v34/commons/app_bar/app_bar_with_image.dart';
+import 'package:v34/commons/favorite/favorite.dart';
+import 'package:v34/commons/text_tab_bar.dart';
+import 'package:v34/models/team.dart';
+import 'package:v34/pages/team-details/results/team_results.dart';
+
+class TeamDetailPage extends StatefulWidget {
+  final Team team;
+
+  const TeamDetailPage({Key key, @required this.team}) : super(key: key);
+
+  @override
+  _TeamDetailPageState createState() => _TeamDetailPageState();
+}
+
+class _TeamDetailPageState extends State<TeamDetailPage> {
+
+  @override
+  void initState() {
```

Comment:
useless! You did nothing except calling super.
Don't take into account this comment, if other PRs change the implementation